plenum/server/client_authn.py

```diff
@@ -100,8 +100,8 @@ def authenticate_multi(self, msg: Dict, signatures: Dict[str, str],
                 raise InvalidSignatureFormat from ex
 
             ser = self.serializeForSig(msg, identifier=idr)
-            verkey = self.getVerkey(idr)
 
+            verkey = self.getVerkey(msg)
```

Comment:
I believe we need to pass both `identifier` and `request` here. Otherwise we will not be able to get verkry in case of multi-sig where `signatures` is a dict of idrs and keys.

Reply:
thnx, fixed